tests/bdd/features/http_workflows.feature

```diff
@@ -0,0 +1,66 @@
+Feature: HTTP Workflow Testing
+  As a workflow developer
+  I want to create and test HTTP-based workflows
+  So that I can build web services and APIs using the workflow engine
+
+  Background:
+    Given the workflow UI is running
+    And I am logged in as an admin user
+
+  Scenario: Create a basic HTTP server workflow
+    When I create an HTTP server workflow with:
+      | address | :8080 |
+      | route   | GET /api/test |
+    Then the workflow should be created successfully
+    And the workflow should use "http.server" module
+    And the workflow should handle HTTP requests
+
+  Scenario: Create HTTP workflow with multiple routes
+    When I create an HTTP server workflow with:
+      | address | :8081 |
+      | route   | GET /api/users |
+      | route   | POST /api/users |
+      | route   | GET /api/health |
+    Then the workflow should be created successfully
+    And the workflow should use "http.router" module
+    And the workflow should handle HTTP requests
+
+  Scenario: Create HTTP workflow with middleware
+    When I create a workflow with:
+      | name        | HTTP Middleware Workflow |
+      | description | HTTP server with auth middleware |
+      | config      | modules:\n  - name: http-server\n    type: http.server\n    config:\n      address: ":8082"\n  - name: auth-middleware\n    type: http.middleware.auth\n    config:\n      authType: "Bearer"\n  - name: api-router\n    type: http.router\n  - name: test-handler\n    type: http.handler\n\nworkflows:\n  http:\n    routes:\n      - method: GET\n        path: /api/secure\n        handler: test-handler\n        middlewares:\n          - auth-middleware |
```

Comment:
All these config scenarios, where the config is inline, would be better off residing in a config scenarios directory, where each config is named for the scenario that it represents. And then that file should be loaded rather than these poorly formatted direct mappings.